README.md

```diff
@@ -92,6 +96,10 @@ You'll need to create a `.env` file in the root of the repo, with the following
 
 `AIRTABLE_API_KEY` - api key for accessing the airtable with youtube video references
 
+`CRM_IMPOSTER_EMAIL` - when using SKIP_AUTH, this is the e-mail used to generate an access cookie
+
+#### Deprecated:
```

Comment:
It looks like these are actually still in use, and not yet deprecated?
`CRM_SIGNING_SECRET`, `CRM_IMPOSTER_ID`, `NYCID_CONSOLE_PASSWORD`

Reply:
I'll review the README and make sure it's up to date!

Reply:
Thanks!!